### PR TITLE
Fix dependency tracing for monorepos with no `package.json` at the repository root

### PR DESCRIPTION
**Pull request by ghengeveld**
<details>
  <summary>📦 Published PR as canary version: <code>7.2.3--canary.827.7535e21.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install chromatic@7.2.3--canary.827.7535e21.0
  # or 
  yarn add chromatic@7.2.3--canary.827.7535e21.0
  ```
</details>

### CODE REVIEW

**Review by ethriel3695 — APPROVED** [review]
This is great @ghengeveld

**Review by weeksling — APPROVED** [review]

**Review by weeksling — COMMENTED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 7.3.0 - 2023-10-02
+
+- [822](https://github.com/chromaui/chromatic-cli/pull/822) Add `experimental_abortSignal` to Node API to allow canceling builds
+- [819](https://github.com/chromaui/chromatic-cli/pull/819) Migrate from Jest to Vitest, update ESLint config and upgrade Execa
```
Comment (weeksling) [inline CHANGELOG.md:4]: It looks like these were merged before. I take it they weren't added to the changelog already?

Comment (ghengeveld) [reply CHANGELOG.md:4]: I'll clean that up.

**Pull request merged**